etcd/getting-started-with-etcd.md

```diff
@@ -44,6 +44,50 @@ etcd v3 startup can be configured at a new node's first boot with [Ignition][ign
 }
 ```
 
+`etcd-member.service` has some default settings
+
+```bash
+$ systemctl cat etcd-member.service
+
+<<COMMENT
+...
+Environment="ETCD_IMAGE_TAG=v3.0.10"
+Environment="ETCD_NAME=%m"
+Environment="ETCD_USER=etcd"
+...
+COMMENT
+```
+
+Default configurations can be overwritten with `1-override.conf` file, as below:
+
+```bash
+# to update etcd-member service file
+$ cat > /tmp/override-my-etcd.conf <<EOF
```

Comment:
No particular reason, I just find it easy to move around those when `sudo` is needed.